openapi/index.yaml

```diff
@@ -92,4 +92,21 @@ definitions:
         $ref: "#/definitions/Timestamp"
     required:
       - expires_at
+  OtpCode:
+    description: The unique otp code that can be use for discounts
+    type: string
+    pattern: "^[A-Z0-9]{9,13}$"
+  Otp:
+    type: object
+    properties:
+      code:
+        $ref: "#/definitions/OtpCode"
+      expires_at:
+        $ref: "#/definitions/Timestamp"
+      ttl:
+        type: number
+    required:
+      - code
+      - expires_at
+      - ttl
```

Comment:
These types are shared with `io-functions-cgn`, can we import them from the source? [ref](https://github.com/pagopa/io-functions-cgn/blob/master/openapi/index.yaml#L465)